bulletroboy/exoforce/exoforce.py

```diff
@@ -331,10 +339,13 @@ def __init__(self, cage_conf, node_name):
 		self.cage = Cage(cage_conf.cage_structure['height'], cage_conf.cage_structure['radius'], self.get_motors())
 
 		self.init_end_effectors()
+		self.init_force_decomp_params()
 
-		self.cage_state_publisher = self.create_publisher(CageState, '/roboy/simulation/cage_state', 1)
+		self.cage_state_publisher = self.create_publisher(CageState, Topics.CAGE_STATE, 1)
 		self.initial_conf_service = self.create_service(GetCageEndEffectors, '/roboy/configuration/end_effectors', self.get_end_effectors_callback)
```

Comment:
why not use here as well Topics.this_name?

Reply:
I think I missed it, because no one is calling that service, will do the change